float2words/float2words.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+from num2words import num2words
+from decimal import Decimal, ROUND_HALF_UP
+
+
+def float2words(number, lang, sfx1='', sfx2='', connector=', ',
+                precision=2):
+    """Convert float to words by given optional parameters.
+
+    Args:
+        number (float): float number.
+        lang (str): language code (e.g. 'en_US', 'ru_RU', etc.).
+        sfx1 (str): suffix for whole part of float number
+            (e.g. 'USD', '$', 'kg', etc.).
+        sfx2 (str): suffix for decimal part of float number.
+            (e.g. 'ct.', 'g', etc.)
+        connector (str): connector will be used between whole and
+            decimal parts expressed in words.
+        precision (int): number of digits after decimal point.
+
+    Returns:
+        number_in_words (str): float number in words.
+
+    """
+    whole_number = int(number // 1)
```

Comment:
Part where you split number into two part could have been moved to its own method: `_split_float` or something. It could be kind of useful on it own, because sometimes we might only need those parts and not to convert it to number. You can make a TODO, to refactor like this.